core/src/main/java/org/eclipse/krazo/core/DefaultViewPathResolver.java

```diff
@@ -0,0 +1,29 @@
+package org.eclipse.krazo.core;
+
+import java.lang.reflect.Method;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Default;
+
+
+/**
+ * Default view path resolver that keeps default compatibility
+ *
+ * @author Jonatan Lemes
+ */
+@Default
+@ApplicationScoped

```

Comment:
I'm not completely sure if we should use CDI for discovery here. The existing Krazo code uses `ServiceLoader` SPIs instead. And IMHO we should be consistent in this regard.